src/bundles/config.js

```diff
@@ -1,3 +1,4 @@
+import memoizee from 'memoizee'
```

Comment:
nit: `p-memoize` is 6KB (vs 266KB of memoizee) 

AFAIK the only place we use memoizee is `/src/bundles/peer-locations.js`, and it does not do anything fancy to justify the size.  Can we switch to the smaller lib?

Reply:
Good find, I changed it :)